chaoss-groups/working-groups.md

```diff
@@ -1,20 +1,51 @@
+
+### Active Working Groups
 * [**Common Metric**](https://github.com/chaoss/wg-common): This working group focuses on defining the metrics that are used by both working groups or are important for community health, but that does not cleanly fit into one of the other existing working groups.
 
 * [**Diversity, Equity & Inclusion**](https://github.com/chaoss/wg-diversity-inclusion): This working group aims to bring together experiences measuring diversity, equity and inclusion in open source projects.
 
-* [**Evolution**](https://github.com/chaoss/wg-evolution): This working group focusses on refining the metrics that inform evolution and to work with open source software implementations.
-
 * [**Risk**](https://github.com/chaoss/wg-risk): The working group focuses on compliance and risk metrics in Open Source Software.
 
-* [**OSPO**](https://github.com/chaoss/wg-ospo): The OSPO Metrics working group aims to advance how organizations understand the value that open source projects can provide as well as the value of these programs / initiatives.
-
 * [**Metrics-Models**](https://github.com/chaoss/wg-metrics-models): This working group develops models that include the integration of multiple CHAOSS metrics in a way that people would consume them in practice.
 
+### Inactive Working Groups
+
+* [**Evolution**](https://github.com/chaoss/wg-evolution): This working group was focused on refining the metrics that inform evolution and to work with open source software implementations.
+
+* [**Value**](https://github.com/chaoss/wg-value): This working group was focused on developing metrics that describe value of open source engagement for individuals and organizations.
+
+
+## 2. Context Working Group
+Context WGs are groups of people who share similar contexts related to open source project health. Context WGs are responsible for helping put CHAOSS metrics, metrics models, and software into practice in meaningful ways across varied contexts.
+
+* [**OSPO**](https://github.com/chaoss/wg-ospo): The OSPO Metrics working group aims to advance how organizations understand the value that open source projects can provide as well as the value of these programs / initiatives.
+
 * [**Application Ecosystem**](https://github.com/chaoss/wg-app-ecosystem): This working group applies CHAOSS metrics in the context of an open-source app ecosystem. The mission of this working group is to build a base set of metrics that is focused on the needs of open source communities that are part of the FOSS app ecosystem.
 
+* [**OSPO++**](https://docs.google.com/document/d/1ZZg8vGQOgfhK6P6G8GQse6OaN3yb2Z_IQVyINa-0TZs/edit): The OSPO++ working group aims create a set of recommendations for metrics related to university open source software, particularly as they relate to what might be seen as “success” for university leadership, faculty, students, and staff, keeping in mind that these actors may evaluate success differently. 
```

Comment:
```suggestion
* [**University**](https://docs.google.com/document/d/1ZZg8vGQOgfhK6P6G8GQse6OaN3yb2Z_IQVyINa-0TZs/edit): The University working group is a joint effort between the CHAOSS project and the OSPO++ community that aims to create a set of recommendations for metrics related to university open source software, particularly as they relate to what might be seen as “success” for university leadership, faculty, students, and staff, keeping in mind that these actors may evaluate success differently. 
```

@germonprez @ElizabethN - we're a bit inconsistent in the naming of this WG between the doc and Slack - does this proposed change seem like the right approach?